config/v1/0000_10_config-operator_01_dns-CustomNoUpgrade.crd.yaml

```diff
@@ -68,13 +68,6 @@ spec:
                         - None
                         - VSphere
                         - oVirt
-                        - IBMCloud
-                        - KubeVirt
-                        - EquinixMetal
-                        - PowerVS
-                        - AlibabaCloud
-                        - Nutanix
-                        - External
```

Comment:
The update automation removed these from the crd YAML's.
I'm not sure if that is valid or not, perhaps I need something configured properly to prevent this removal from happening (via `make update`).

Reply:
I see PR tests failed because of these weird changes
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_api/1499/pull-ci-openshift-api-master-verify/1679175554709852160

So, I'm going to manually remove/revert those changes and assume something locally is causing the problem (since CI doesn't expect those changes).

Reply:
I resolved the issue and re-generated the files properly.
Tests passed now.